esmvalcore/cmor/_fixes/access/access_esm.py

```diff
@@ -0,0 +1,128 @@
+"""On-the-fly CMORizer for ACCESS-ESM.
+
+Note
+----
+This is the first version of ACCESS-ESM CMORizer in for ESMValCore
+Currently, only two variables (`tas`,`pr`) is fully supported.
+"""
+import logging
+import os
+
+from iris.cube import CubeList
+
+from esmvalcore.cmor._fixes.native_datasets import NativeDatasetFix
+
+logger = logging.getLogger(__name__)
+
+
+class Tas(NativeDatasetFix):
+    """Fix variable(tas) only."""
+
+    def __init__(self, vardef, extra_facets, session, frequency):
+        """Initialise some class variable Heritage from native_dataset."""
+        super().__init__(vardef, extra_facets, session, frequency)
+
+        self.cube = None
+
+        self.current_dir = os.path.dirname(__file__)
+
+    def fix_height_name(self):
+        """Fix variable name of coordinate 'height'."""
+        if self.cube.coord('height').var_name != 'height':
+            self.cube.coord('height').var_name = 'height'
+
+    def fix_long_name(self):
+        """Fix variable long_name."""
+        self.cube.long_name = 'Near-Surface Air Temperature'
+
+    def fix_var_name(self):
+        """Fix variable long_name."""
+        self.cube.var_name = 'tas'
+
+    def fix_coord_system(self):
+        """Delete coord_system to make CubeList able to merge."""
+        for dim in self.cube.dim_coords:
+            if dim.coord_system is not None:
+                self.cube.coord(dim.standard_name).coord_system = None
+
+    def fix_metadata(self, cubes):
+        """Fix metadata.
+
+        Fix name of coordinate(height), long name and variable name of
+        variable(tas).
+
+        Parameters
+        ----------
+        cubes : iris.cube.CubeList
+            Input cubes.
+
+        Returns
+        -------
+        iris.cube.CubeList
+        """
+        original_short_name = 'fld_s03i236'
```

Comment:
Please add this information to an extra facet file ([example](https://github.com/ESMValGroup/ESMValCore/blob/main/esmvalcore/config/extra_facets/emac-mappings.yml)) as `raw_name`. With this, the `NativeDatasetFix.get_cube` function that you're using below will automatically find the correct variable.